src/components/Button/Button.scss

```diff
@@ -2,25 +2,36 @@
 @import "~office-ui-fabric/src/components/Button/Button";
 
 .ms-Button--icon {
+  @include focus-border(0px);
   background-color: transparent;
+  color: $ms-color-neutralSecondary;
   padding: 0;
   min-width: auto;
   height: auto;
   border: 0;
 
-  &:hover {
+  &:hover,
+  &:active {
     background-color: transparent;
     border-color: transparent;
     outline-color: transparent;
+    color: $ms-color-neutralPrimary;
   }
```

Comment:
Please verify this still looks like a button or how you'd expect in high contrast.